nuclide/atom-test-runners/jasmine-runner.js

```diff
@@ -0,0 +1,15 @@
+const { createRunner } = require("atom-jasmine3-test-runner")
+
+const fs = require("fs")
+
+// https://github.com/UziTech/atom-jasmine3-test-runner#api
+module.exports = createRunner({
+  testPackages: process.env.APM_TEST_PACKAGES ? process.env.APM_TEST_PACKAGES.split(' ') : [],
+  suffix: "-test-v3",
+  legacySuffix: "-test",
+  testPaths: ["__tests__", "__atom_tests__"].filter(dir => !fs.existsSync(process.cwd(), dir)),
```

Comment:
`!fs.existsSync` filters the paths that dont exist.

and [existsSync](https://nodejs.org/api/fs.html#fs_fs_existssync_path) only takes one parameter. This should be `dir => fs.existsSync(path.resolve(dir))`

Reply:
Thanks. I did not have enough focus while I wrote this.